staff/marti-malek/e-commerce-protos/scripts/boxer.js

```diff
@@ -0,0 +1,6 @@
+function Boxer() {
```

Comment:
Why is it empty??

Reply:
It is empty because it was not needed in the demo, so no item of this type was created in the first place.